gwt-jackson/src/main/java/com/github/nmorel/gwtjackson/client/deser/BaseDateJsonDeserializer.java

```diff
@@ -57,11 +56,13 @@ protected Date deserializeNumber( long millis, JsonDeserializerParameters params
         }
 
         @Override
-        protected Date deserializeString( String date, JsonDeserializerParameters params ) {
-            return DateFormat.parse( params.getPattern(), date );
+        protected Date deserializeString( String date, JsonDeserializationContext ctx, JsonDeserializerParameters params ) {
+            return DateFormat.parse(ctx.isAdjustDatesToContextTimeZone(), params.getPattern(), date );
         }
     }
 
+    private static final String SQL_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
Can you move the format inside the inner class `SqlDateJsonDeserializer` please ?